Makefile

```diff
@@ -49,7 +49,7 @@ battletest: ## Run randomized, racing, code coveraged, tests
 
 e2etests: ## Run the e2e suite against your local cluster
 	go clean -testcache
-	go test -p 1 -timeout 180m -v ./test/suites/... -run=${TEST_FILTER}
+	CLUSTER_NAME=${CLUSTER_NAME} go test -p 1 -timeout 180m -v ./test/suites/... -run=${TEST_FILTER}
```

Comment:
Makefile doesn't do an export if it sets the variable on your behalf, meaning that it will not be set in the sub-process launched by e2etests if unset in the parent process.